controllers/reviewsController.js

```diff
@@ -17,11 +14,33 @@ exports.getReviews = (request, response, next) => {
 
 exports.getReviewById = (request, response, next) => {
   const { review_id } = request.params;
+
   fetchReviewById(review_id)
     .then((review) => {
       response.status(200).send({ review });
     })
     .catch((error) => {
+      console.log(error);
       next(error);
     });
-};
\ No newline at end of file
+};
+
+exports.patchReviewById = (request, response, next) => {
+  const { inc_votes } = request.body;
+  const { review_id } = request.params;
+
+  fetchReviewById(review_id)
+    .then((review) => {
+      if ( inc_votes === undefined) {
+        return Promise.reject('Property not found!')
+      }
+      review.votes += inc_votes;
+      if (Math.sign(inc_votes) === -1 && review.votes < 0) {
```

Comment:
I think this bit of code stops your votes from going into negative numbers? Just remember where this is for when you get to frontend - you may find that you want to be able to display negative votes on a review and if so, this will stop you from being able to do that. It's entirely up to you though :)